content/about/student_stories.rst

```diff
@@ -25,6 +25,61 @@ basics and saw how easy it was to add their laptops to the cluster as a diskless
 node, and also gained an appreciation of open source software.
 
 
+Cody Holliday on the Department of Energy Cyber Defense Competition 2018
+------------------------------------------------------------------------
+4/23/18
```

Comment:
For some reason my old review got outdated so I'm recreating it here.

This still needs a newline before the date to fix formatting, at least to keep the format between the student stories the same, since all the others have a newline there.

Reply:
Oh, so you want a newline BEFORE the date. I'll go ahead and do that!